packages/shared/lib/core/wallet/classes/activity.class.ts

```diff
@@ -217,6 +221,9 @@ export class Activity implements IActivity {
 
     getTitle(): string {
         let title = ''
+        if (this.type === ActivityType.Minting) {
+            title = 'general.minting'
+        }
```

Comment:
This should also be based on the inclusion state so it says minting only while pending